tests/plugins/test_tlctr.py

```diff
@@ -8,3 +8,7 @@ class TestPluginCanHandleUrlTLCtr(PluginCanHandleUrl):
     should_match = [
         'https://www.tlctv.com.tr/canli-izle',
     ]
+
+    should_not_match = [
+        'https://example.com/index.html',
+    ]
```

Comment:
Generic negative URL tests are already added by the `PluginCanHandleUrl` base class.